network/src/network.rs

```diff
@@ -447,6 +447,41 @@ pub struct EventHandler {
     pub(crate) network_state: Arc<NetworkState>,
 }
 
+impl EventHandler {
+    fn inbound_eviction(&self, context: &mut ServiceContext) {
+        if self.network_state.config.bootnode_mode {
+            let status = self.network_state.connection_status();
+
+            if status.max_inbound
+                - status.unreserved_inbound
+                - self.network_state.config.reserved_peers.len() as u32
+                <= 10
```

Comment:
It assumes that `max_peers` is large(should be greater than 10 * 2).
What if I wanna set the `max_peers  = reserved_peers.len()`, in other words, I want to ensure that only connect to the reserved peers?

Reply:
Suggest change to avoid overflow panic:

```
 if status.max_inbound <=
                status.unreserved_inbound
                + self.network_state.config.reserved_peers.len() as u32
                + 10
```

Reply:
I found another overflow bug...